pkg/services/ngalert/api/api.go

```diff
@@ -149,4 +151,10 @@ func (api *API) RegisterAPIEndpoints(m *metrics.API) {
 		logger: logger,
 		hist:   api.Historian,
 	}), m)
+
+	api.RegisterUpgradeApiEndpoints(NewUpgradeApi(NewUpgradeSrc(
+		logger,
+		api.UpgradeService,
+		api.Cfg,
+	)), m)
```

Comment:
Probably skip registering these routes if UA is enabled? I see there is some code in request handler that checks it and that can be removed if we just do not register routes.